indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/ec2/EC2AutoScaler.java

```diff
@@ -70,6 +72,13 @@ public EC2AutoScaler(
     this.envConfig = envConfig;
     this.amazonEC2Client = amazonEC2Client;
     this.config = config;
+
+    // Set the AWS region to work in according to envConfig, if set.
+    try {
+      amazonEC2Client.setRegion(RegionUtils.getRegion(envConfig.getRegion()));
```

Comment:
Suggest taking region from config and simply skipping this step if it is null. Doing so retains prior behavior.


```java
final String region = envConfig.getRegion();
if(region != null) {
  amazonEC2Client.setRegion(RegionUtils.getRegion(region));
}
```